From 4e9f4e6136a7c35b0b3d341424eb4e0585f2f0a7 Mon Sep 17 00:00:00 2001
From: Jian-Hong Pan <jian-hong@endlessm.com>
Date: Wed, 28 Nov 2018 17:06:29 +0800
Subject: [PATCH] XF86keysym: Add XF86XK_MonBrightnessCycle

Add XF86XK_MonBrightnessCycle keysym, to be used as mapping for evdev's
KEY_BRIGHTNESS_CYCLE keycode which is generated from ACPI video module's
ACPI_VIDEO_NOTIFY_CYCLE_BRIGHTNESS on some Acer AIO desktop buttons.
The button changes the screen's brightness on Windows.

Fixes: https://bugs.freedesktop.org/show_bug.cgi?id=108861
Signed-off-by: Jian-Hong Pan <jian-hong@endlessm.com>
Reviewed-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 include/X11/XF86keysym.h | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/include/X11/XF86keysym.h b/include/X11/XF86keysym.h
index 771fbb0..9ad8948 100644
--- include/X11/XF86keysym.h
+++ include/X11/XF86keysym.h
@@ -23,11 +23,12 @@
 #define XF86XK_ModeLock		0x1008FF01	/* Mode Switch Lock */
 
 /* Backlight controls. */
-#define XF86XK_MonBrightnessUp   0x1008FF02  /* Monitor/panel brightness */
-#define XF86XK_MonBrightnessDown 0x1008FF03  /* Monitor/panel brightness */
-#define XF86XK_KbdLightOnOff     0x1008FF04  /* Keyboards may be lit     */
-#define XF86XK_KbdBrightnessUp   0x1008FF05  /* Keyboards may be lit     */
-#define XF86XK_KbdBrightnessDown 0x1008FF06  /* Keyboards may be lit     */
+#define XF86XK_MonBrightnessUp    0x1008FF02  /* Monitor/panel brightness */
+#define XF86XK_MonBrightnessDown  0x1008FF03  /* Monitor/panel brightness */
+#define XF86XK_KbdLightOnOff      0x1008FF04  /* Keyboards may be lit     */
+#define XF86XK_KbdBrightnessUp    0x1008FF05  /* Keyboards may be lit     */
+#define XF86XK_KbdBrightnessDown  0x1008FF06  /* Keyboards may be lit     */
+#define XF86XK_MonBrightnessCycle 0x1008FF07  /* Monitor/panel brightness */
 
 /*
  * Keys found on some "Internet" keyboards.
-- 
2.18.1


